.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,68 @@
+name: trivy  ⏲
```

Comment:
Call this `Security Scan`. Also rename the workflow to `security-scan.yaml`

Reply:
done